BlackJackets API/src/main/java/BlackJackets/BlackJackets/dto/MessageDTO.java

```diff
@@ -0,0 +1,31 @@
+package BlackJackets.BlackJackets.dto;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.*;
+
+@Entity
```

Comment:
The other DTOs do not have @Entity. Should this be removed?

Reply:
done